behave/__main__.py

```diff
@@ -215,7 +215,7 @@ def main(args=None):
     :return: 0, if successful. Non-zero, in case of errors/failures.
     """
     config = Configuration(args)
-    return run_behave(config)
+    return run_behave(config, runner_class=config.runner_class)
```

Comment:
I see, this would be a one-liner conflict with parallel, can fix directly. So, all clear here. :)

Reply:
Can you explain a little bit more what you mean about "conflict with parallel"? and why it's not in the tests? O:)

Reply:
NM. https://github.com/behave/behave/pull/616/files#diff-c11353c15368f2721090f13f46563e3cR209 i understand now.